build.gradle.kts

```diff
@@ -23,7 +23,7 @@ object Versions {
     const val logstashEncoder = "7.0.1"
     const val mockk = "1.12.2"
     const val nimbusjosejwt = "9.18"
-    const val postgresEmbedded = "0.13.4"
+    const val postgresEmbedded = "1.0.0"
```

Comment:
Glemt å fjerne denne?

Reply:
Jeg hadde ekstremt problemer med å få postgresEmbedded til å fungere lokalt. Er det ikke like så greit å oppgradere til siste versjon?

Reply:
Nei, for 1.0.0 fungerer ikke uten videre på windows lokalt. Derimot, fungerer 0.3.14 helt fint(på linux, hos og flere andre.)

Hva er det som ikke fungerer hos deg?